<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diferencia</class>
 <widget class="QWidget" name="Diferencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>341</width>
     <height>173</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxhResult">
      <property name="text">
       <string>Ver histograma resultante (imagen Diferencia)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxhSelect">
      <property name="text">
       <string>ver histograma de la imagen seleccionada (I2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>n (Umbral de cambios):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxN"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxCambios1">
      <property name="text">
       <string>Ver cambios de la imagen original (I1)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxCambios2">
      <property name="text">
       <string>Ver cambios de la imagen seleccionada (I2)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ButtonCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonAbrir">
      <property name="text">
       <string>Abrir otra</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonAceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Está bien esta imagen?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
